personality/se_dpm/service_puppet.pan

```diff
@@ -31,16 +33,15 @@ include 'personality/se_dpm/puppet/puppetconf';
 
 include 'personality/se_dpm/rpms/config';
 
-'/software/packages' = {
-    if (SEDPM_IS_HEAD_NODE) {
-        if (match(OS_VERSION_PARAMS['major'], '[es]l[56]')) {
-            pkg_repl('mysql-server');
-        } else {
-            pkg_repl('mariadb-server');
-        };
+# Configure and enable MySQL server
+variable DPM_MYSQL_INCLUDE = {
+    if (SEDPM_IS_HEAD_NODE && (FULL_HOSTNAME == DPM_MYSQL_SERVER)) {
+        'personality/se_dpm/puppet/mysql';
+    } else {
+        null;
     };
-    SELF;
 };
+include DPM_MYSQL_INCLUDE;
```

Comment:
Since dpm puppet modules already configure mysql can this include be opt-in (or at least opt-out). For example with a DPM_MYSQL_CONFIGURE boolean variable?

Reply:
You are right, we should add a switch that is set to no by default to ensure backward compliance.